Monika After Story/game/script-topics.rpy

```diff
@@ -1005,42 +999,40 @@ label monika_sayori:
     m 2lsc "I still wish I could have handled that whole thing a little more tactfully."
 
     if (
-            not persistent._mas_sensitive_mode
-            and not renpy.seen_label("monika_sayori")
-            and not persistent._mas_pm_cares_about_dokis
+            not renpy.seen_label("monika_sayori")
```

Comment:
```suggestion
            not mas_getEVL_shown_count("monika_sayori")
```
The label is marked as seen the moment we get in so we should check the `shown_count` in this one